src/Components/Web.JS/src/Platform/Circuits/RenderQueue.ts

```diff
@@ -20,14 +18,6 @@ export class RenderQueue {
     this.logger = logger;
   }
 
-  public static getOrCreate(logger: Logger): RenderQueue {
-    if (!RenderQueue.instance) {
-      RenderQueue.instance = new RenderQueue(logger);
-    }
-
-    return this.instance;
-  }
-
```

Comment:
It doesn't make sense to make `RenderQueue` a singleton anymore because there may be more than once circuit active within the lifetime of the document. It's easier to construct a `RenderQueue` upon circuit initialization rather than having to remember to reset the existing `RenderQueue`'s state when a new circuit gets initialized.